test/extended/clusterversion/clusterversion.go

```diff
@@ -0,0 +1,46 @@
+package clusterversion
+
+import (
+	"context"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-cluster-lifecycle][OCPFeatureGate:ImageStreamImportMode] ClusterVersion API", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("")
+
+	g.It("desired.architecture field in the CV should be multi or empty depending on the 'release.openshift.io/architecture' field being set in the release payload's metadata [apigroup:config.openshift.io]", func() {
+		TestClusterVersionDesiredArchitecture(g.GinkgoT(), oc)
+	})
+
+})
+
+func TestClusterVersionDesiredArchitecture(t g.GinkgoTInterface, oc *exutil.CLI) {
+	ctx := context.Background()
+	if !exutil.IsTechPreviewNoUpgrade(ctx, oc.AdminConfigClient()) {
+		g.Skip("skipping, this feature is only supported on TechPreviewNoUpgrade clusters")
```

Comment:
I don't think we want this guard.  The `OCPFeatureGate:ImageStreamImportMode` naming should already gate this test-case to clusters where `ImageStreamImportMode` is enabled.  That's _currently_ in tech-preview clusters, but the feature-gate will eventually GA, and we want the origin suite to gracefully understand that transition and begin running the test-case against those GA clusters, without us having to come back in and remove this skip then.

Reply:
makes sense. changed.